README.rst

```diff
@@ -120,16 +120,16 @@ The plugin itself holds a default config that can be used for any Coverity proje
         'stream': 'some_coverty_stream',
     }
 
-This default configuration build into the plugin, can be overriden through the conf.py of your project.
+This default configuration, built into the plugin, can be overridden through the *conf.py* of your project.
 
 
 -----
 Usage
 -----
 
-Inside your reStructured text file you can call a block `.. coverity-list:` which will generate the table
-with title and defined columns. For example to display CID, Classification, Action and Comment columns while
-filtering classification items with value `Bug` you should use the following snippet:
+Inside your reStructured text file, you can call a block `.. coverity-list:` which will generate the table
```

Comment:
I would put comma here before which not where you put it.

Reply:
24ff2f805b9c3789c10252c0f528d871ab0face2